kinova_tracking/pos_kinova_arm_2.py

```diff
@@ -13,19 +13,18 @@
 import random
 
 
-def move_marker_table(labels_markers_list: list[str], c3d_file: c3d, offset: float) -> np.array:
+def move_marker(marker_to_move: str, c3d_file: c3d, offset: np.ndarray, ) -> np.array:
```

Comment:
int or str ??

index instead of indices

Reply:
It's a int